third_party/llvm-project/libcxxabi/BUILD.gn

```diff
@@ -65,11 +64,19 @@ static_library("cxxabi") {
     "src/cxa_aux_runtime.cpp",
     "src/cxa_default_handlers.cpp",
     "src/cxa_demangle.cpp",
-    "src/cxa_exception.cpp",
+
+    # Requires exception support
```

Comment:
Do we ever expect a platform/experiment to want these files with exceptions to be compiled? If so, we should follow the pattern the upstream sets in https://github.com/llvm/llvm-project/blob/main/libcxxabi/src/CMakeLists.txt#L28

Reply:
No, we don't use exceptions:
https://google.github.io/styleguide/cppguide.html#Exceptions